layers/+lang/java/packages.el

```diff
@@ -347,7 +347,7 @@
 (defun java/post-init-counsel-gtags ()
   (spacemacs/counsel-gtags-define-keys-for-mode 'java-mode))
 
-(defun java/post-init-helm-gtags ()
+(defun ""java/post-init-helm-gtags ()
```

Comment:
This change breaks the java layer

Reply:
fixed, no idea how that got there, I did not touch the java layer